.github/ISSUE_TEMPLATE/general.md

```diff
@@ -0,0 +1,9 @@
+---
+name: General
+about: Leave a general comment
```

Comment:
I would maybe change this to "Leave a general comment or question" so that it can also cover support questions.